packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -329,6 +332,21 @@ class _InspectorTreeState extends State<InspectorTree>
     return maxOffset - viewportDimension;
   }
 
+  void _handleKeyEvent(RawKeyEvent event) {
+    if (event is! RawKeyDownEvent) return;
+
+    // Exit early if we aren't handling the key
```

Comment:
This comment doesn't seem to match the code below it. Can we just remove it?

Reply:
done.